bin/processSppRdata.r

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env Rscript
+
+# R scripts for convert the Rdata output file from phantompeakqualtools for strand-shift plot
+# Author @chuan-wang https://github.com/chuan-wang
+
+# Command line argument processing
+args <- commandArgs(trailingOnly=TRUE)
+
+# Check input args
+if (length(args) != 2) {
+  stop("Usage: processSppRdata.r [ cross-correlation.Rdata ] [ cross-correlation.csv ]", call.=FALSE)
```

Comment:
Maybe `processSppRdata.r [ input.Rdata ] [ output.csv ]` instead? I'd be confused as to how to supply the CSV input otherwise.